src/cli/index.js

```diff
@@ -212,9 +179,55 @@ const questions = {
       },
     },
   ],
-};
+});
 
 async function run() {
+  let appPath = appPathFromArgument;
+  if (!appPath) {
+    const answers = await inquirer.prompt([
+      {
+        type: 'input',
+        name: 'appPath',
+        message: 'Project directory:',
+      },
+    ]);
+    appPath = answers.appPath;
+  }
+  if (appPath.startsWith('~/')) {
```

Comment:
Unlike command-line argument, the string path including `~/` needs to be expanded by ourselves.

Reply:
How does it work if it starts with `/` or `./`? Perhaps we should use a Regex instead.

Reply:
is there a way to forward the path expanding to the shell instead?

Reply:
If it's a problem, there's also https://nodejs.org/api/path.html#path_path_relative_from_to

Reply:
@francoischalifour @Haroenv 
In the end, it gets resolved [here](https://github.com/algolia/create-instantsearch-app/blob/d5a5655c2c284eb43499aad6c490b95a2b7b4b0c/src/api/index.js#L19).

```
> path.resolve('/test')
'/test'
> path.resolve('./test')
'/Users/eunjaelee/workspace/create-instantsearch-app/test'
> path.resolve('~/test')
'/Users/eunjaelee/workspace/create-instantsearch-app/~/test'
> 
```

If a path starts with `/` or `./`, `path.resolve()` works correctly. However `~` didn't work, because, I guess, it has no context of home directory, or something.

Reply:
Thanks for checking Eunjae!